mavros_extras/src/plugins/mount_status.cpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @brief Mouny Status plugin
+ * @file mount_status.cpp
+ * @author André Ferreira <andre.ferreira@beyond-vision.pt>
+ *
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2021 André Ferreira.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#include <mavros/mavros_plugin.h>
+
+#include <mavros_msgs/MountStatus.h>
+
+namespace mavros {
+namespace extra_plugins {
+/**
+ * @brief Mount Status plugin
+ *
+ * Publish Mount status from FCU.
+ * @see command_cb()
+ */
+class MountStatusPlugin : public plugin::PluginBase {
+public:
+	MountStatusPlugin() : PluginBase(),
+		nh("~"),
+		mount_nh("~mount_status")
+	{ }
+
+	void initialize(UAS &uas_) override
+	{
+		PluginBase::initialize(uas_);
+
+		mount_status_pub = mount_nh.advertise<mavros_msgs::MountStatus>("orientation", 10);
```

Comment:
Orientation usually means that you provide quaternion. I think it's bad name for something custom.